streams/de.m3u

```diff
@@ -140,11 +138,11 @@ https://ub1doy938d.gjirafa.net/live/oEZi1otQqIHglY0tzZidyCPWhxrS5WgO/yky0y1.m3u8
 http://51.15.2.151/hls/kanalavrupa.m3u8
 #EXTINF:-1 tvg-id="KaufBeiTV.de",KaufBei TV (576p)
 https://api.alpaca.t62a.com/hls/9103/index.m3u8
-#EXTINF:-1 tvg-id="KIKA.de",KiKA [Geo-blocked]
+#EXTINF:-1 tvg-id="KIKA.de",KiKA (1080p) [Geo-blocked]
 https://kikageohls.akamaized.net/hls/live/2022693/livetvkika_de/master.m3u8
```

Comment:
There's also a worldwide version of Kika, heavily censored for copyright reasons but exists : https://kikahls.akamaized.net/hls/live/2022690/livetvkika_ww/master.m3u8

Reply:
Added. Thx bro